tests/manage/z_cluster/test_get_requests_by_kube_probe.py

```diff
@@ -0,0 +1,30 @@
+import logging
```

Comment:
@OdedViner Shouldn't this test be under the UI directory?

Reply:
@PrasadDesala We test it in `test_validation_ui`.
https://github.com/red-hat-storage/ocs-ci/blob/master/tests/ui/test_validation_ui.py#L35